resources/build/build-help.inc.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#
+# This script contains utilities for converting markdown help documentation to htm/html
```

Comment:
Might be good to mention that this script converts to html which will be installed together with the app and used to display help in the app.